replit_river/server_transport.py

```diff
@@ -113,11 +113,11 @@ async def websocket_closed_callback() -> None:
     async def _establish_handshake(
         self, request_message: TransportMessage, websocket: WebSocketCommonProtocol
     ) -> Tuple[
-        ControlMessageHandshakeRequest,
+        ControlMessageHandshakeRequest[Any],
```

Comment:
`RiverConcreteType` instead?

Reply:
This is in the runtime, not the codegen, this is just codifying the previous behavior so none of the consumers break

Reply:
gotcha, cool w me